src/struct.h

```diff
@@ -12,6 +12,8 @@
 
 namespace bpftrace {
 
+static constexpr std::string retval_field_name = "$retval";
```

Comment:
(food for though) We have a mix of constexpr names that are all caps and ones that are like this. What do you think about making file level consts all caps as a style rule?

Reply:
Yeah, makes sense, let me change this one.